.github/workflows/main.yml

```diff
@@ -0,0 +1,55 @@
+name: ci
+
+on:
+  pull_request:
+    branches:
+      - newsite
+
+  push:
+    branches:
+      - newsite
+
+jobs:
+  test:
+    name: Test run
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os:
+          - macOS-latest
+          - ubuntu-latest
+          - windows-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: run test
+        env:
+          REF: ${{ github.ref}}
+        run: |
+          echo Checking github env variables:
```

Comment:
This has to be removed and proper test should run (similar to the travis CI/CD)

Reply:
We don't really have tests. Maybe it can be checked that the Hugo build finishes without errors or warnings?

Reply:
I have removed this job for now. We can add jobs later as per the requirements.